.github/workflows/build.yml

```diff
@@ -56,12 +58,12 @@ jobs:
           rm -rf firedrake_venv
       - name: Build Firedrake
         run: |
+          unset PETSC_DIR PETSC_ARCH SLEPC_DIR
```

Comment:
Why are these being unset?

Reply:
In which case, it all looks good to me. I can approve once the workflow is reverted and tests are passing.

Reply:
Now that I have some written approval I will probably just force things through (assuming things pass) so I don't need to bother you with extra work. Thanks!